tests/java-db-test/src/main/java/testing/servlets/TestServlet.java

```diff
@@ -0,0 +1,66 @@
+package testing.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.sql.PreparedStatement;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * This servlet is just to test a query on the mintern database, and can be used as reference.
```

Comment:
Name of database should be UpperCamelCase.

Reply:
done